Easydict/NewApp/Feature/Shortcut/Shortcut.swift

```diff
@@ -0,0 +1,218 @@
+                            keyCombo: keyCombo,
+                            target: Shortcut.shared,
+                            action: #selector(Shortcut.inputTranslate))
+        case .snipTranslate:
+            hotKey = HotKey(identifier: type.rawValue,
+                            keyCombo: keyCombo,
+                            target: Shortcut.shared,
+                            action: #selector(Shortcut.snipTranslate))
+        case .selectTranslate:
+            hotKey = HotKey(identifier: type.rawValue,
+                            keyCombo: keyCombo,
+                            target: Shortcut.shared,
+                            action: #selector(Shortcut.selectTextTranslate))
+        case .silentScreenshotOcr:
+            hotKey = HotKey(identifier: type.rawValue,
+                            keyCombo: keyCombo,
+                            target: Shortcut.shared,
+                            action: #selector(Shortcut.screenshotOCR))
+        case .showMiniWindow:
+            hotKey = HotKey(identifier: type.rawValue,
+                            keyCombo: keyCombo,
+                            target: Shortcut.shared,
+                            action: #selector(Shortcut.showMiniFloatingWindow))
+        }
+        hotKey.register()
+    }
+}
+
+// shortcut binding func
+extension Shortcut {
+    @objc func selectTextTranslate() {
+        EZWindowManager.shared().selectTextTranslate()
+    }
+
+    @objc func snipTranslate() {
+        EZWindowManager.shared().snipTranslate()
+    }
+
+    @objc func inputTranslate() {
+        EZWindowManager.shared().inputTranslate()
+    }
+
+    @objc func showMiniFloatingWindow() {
+        EZWindowManager.shared().showMiniFloatingWindow()
+    }
+
+    @objc func screenshotOCR() {
+        EZWindowManager.shared().screenshotOCR()
+    }
+}
+
+// fetch shortcut KeyCombo
+extension Shortcut {
+    public func shortcutKeyCombo(_ type: ShortcutType) -> KeyCombo? {
+        switch type {
+        case .inputTranslate:
+            guard let keyCombo = restoreInputTranslate() else { return nil }
+            return keyCombo
+        case .snipTranslate:
+            guard let keyCombo = restoreSnipShortcutKey() else { return nil }
+            return keyCombo
+        case .selectTranslate:
+            guard let keyCombo = restoreSelectionShortcutKey() else { return nil }
+            return keyCombo
+        case .silentScreenshotOcr:
+            guard let keyCombo = restoreScreenshotOCRShortcutKey() else { return nil }
+            return keyCombo
+        case .showMiniWindow:
+            guard let keyCombo = restoreShowMiniWindow() else { return nil }
+            return keyCombo
+        }
+    }
+}
+
+/// can't using keyEquivalent and EventModifiers in SwiftUI MenuItemView direct, because item
+/// keyboardShortcut not support double modifier key but can use ⌥ as character
+extension View {
+    public func keyboardShortcut(_ type: ShortcutType) -> some View {
```

Comment:
`AnyView` is not recommended. Try to replaced with using `@ViewBuilder` modifier. 

```swift
@ViewBuilder
func keyboardShortcut(_ type: ShortcutType) -> some View {
        if let keyCombo = ... {
                self.keyboardShortcut(...)
        } else {
                self
        }
}
```